File: fix_spanish.py
#!/usr/bin/env python3
# Script to fix Spanish FAQ1

with open('/src/locales/translations.ts', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace the Spanish FAQ1 answer
old_text = 'ChefNet es una plataforma global de FoodTech que unifica descubrimiento, reservas, pagos y engagement de restaurantes y chefs privados.'
new_text = 'ChefNet es una plataforma global de FoodTech que unifica el descubrimiento, reservas, pagos y engagement con restaurantes y chefs privados.'

content = content.replace(old_text, new_text)

with open('/src/locales/translations.ts', 'w', encoding='utf-8') as f:
    f.write(content)

print("Fixed Spanish FAQ1!")

File: fix_spanish.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(process.cwd(), 'src', 'locales', 'translations.ts');

// Read file
let content = fs.readFileSync(filePath, 'utf8');

// Old text
const old = 'ChefNet es una plataforma global de FoodTech que unifica descubrimiento, reservas, pagos y engagement de restaurantes y chefs privados.';

// New text  
const newText = 'ChefNet es una plataforma global de FoodTech que unifica el descubrimiento, reservas, pagos y engagement con restaurantes y chefs privados.';

// Replace
if (content.includes(old)) {
    content = content.replace(old, newText);
    fs.writeFileSync(filePath, content, 'utf8');
    console.log('✓ Fixed Spanish FAQ1!');
} else {
    console.log('✗ Old text not found!');
    process.exit(1);
}
